Neos.Flow/Tests/Functional/Mvc/Fixtures/Controller/ActionControllerTestAController.php

```diff
@@ -65,6 +65,15 @@ public function fourthAction($emailAddress)
         $this->view->assign('emailAddress', $emailAddress);
     }
 
+    /**
+     * Tests response and request
+     */
+    public function fifthAction()
+    {
+        $this->response->setHttpHeader('X-Foo', 'Hello World');
```

Comment:
this `$this->response->setHttpHeader` doesnt work in beta14 so id say merge?

cc @kitsunet